### ISSUE TITLE
leisure=nature_reserve boundaries interfer badly with roads

### ISSUE THREAD

**Issue by polarbearing** [gravitystorm/openstreetmap-carto#1199]
Look at these plenty of leisure=nature_reserve in a built-up area:
https://www.openstreetmap.org/way/29837903#map=16/42.1606/-71.1335
(additional tags landuse=conservation or landuse=forest).
1. The darker-green boundaries interfer badly with roads they overlap or touch.
2. The boundary between two polygons of this kind becomes quite strong. 
3. Each of the polygon seems to have a little 'blurb' extending the boundary,
   look at the south-west corner of the example way. It is not the node where the OSM-polygon
   closes, but maybe where carto starts drawing. 
   Apparently it is always the most-western point of the polygon.

Rendering of leisure=nature_reserve was already discussed/changed here #200 #563 #1147

**Comment by mkoniecz** [comment]
ad 3 - a known upstream issue, see #481

**Comment by math1985** [comment]
To me it doesn't look that bad to be honest? Perhaps we could make the boundary a bit lighter?

**Comment by kocio-pl** [comment]
For me too - yes, probably lightening a bit would do everything we need.

**Comment by mkoniecz** [comment]
I think that it is not so bad. And it is a problem of using the same style for stamp-sized nature reserves and giant protected areas.

Maybe some way_area magic would be a good idea?

Also, it is important to keep border noticeable in forests.

**Comment by polarbearing** [comment]
It was mostly the green boundary rendered on top of roads that was hurting my eye.

**Comment by math1985** [comment]
#69 was the reason for this rendering in the first place.

**Comment by math1985** [comment]
> It was mostly the green boundary rendered on top of roads that was hurting my eye.

Seems at least partially caused by bag tagging, unless Pleasant Street is part of the nature reserve: https://www.openstreetmap.org/#map=18/42.16902/-71.13431

**Comment by mkoniecz** [comment]
> #69 was the reason for this rendering in the first place.

Now it is much better, but still not perfect. I have no idea how it may be improved.

**Comment by polarbearing** [comment]
> Seems at least partially caused by bag tagging

This is correct, there seems to be a lack of local mappers in the area, roads and nature are all from imports that need a lot of fine-tuning. I was armchairing there only to fix some vandalism.

**Comment by pnorman** [comment]
This appears to be an issue with a bad import - it wanders across the road, and looks like unmodified MASSGIS data, not normal mapping.

**Comment by jeisenbe** [comment]
Reopened - at mid and low zoom levels this is an issue even when the boundary is properly mapped, and in some places there are protected area boundaries which directly follow roads.
